src/app/examples/doughnut-chart-ns-example/doughnut-chart-ns-example.component.ts

```diff
@@ -19,49 +19,45 @@ export class Distribution {
 })
 
 export class DoughnutChartNsExampleComponent implements OnInit {
-  private chartModels$: Observable<ChartModel[]>;
+  public chartModels$: Observable<ChartModel[]>;
   private _paletteColors = ['#33a87f', '#005d3c', '#02f5a1', '#24765a'];
-  buttonText = 'Se mere';
-  title = 'Investeringsoverblik';
-  subtitle = '';
+  buttonText = 'View details';
+  title = 'Yours investments';
 
   constructor() { }
 
   ngOnInit() {
-    this.chartModels$ = this.convertToChartModel(of(this.getDistributionsMock()).pipe(delay(1000)), this._paletteColors);
+    this.chartModels$ = this.convertToChartModel(this.getMockData(), this._paletteColors).pipe(delay(500));
   }
 
-  getDistributionsMock(): Distribution[] {
+  getMockData(): Distribution[] {
```

Comment:
Change to `getMockData(): [{type: string, drawingPercentage: number, labelPercentage: string}] {`